src/main/java/frc/utils/ChargeStationStates.java

```diff
@@ -0,0 +1,127 @@
+package frc.utils;
+
+import edu.wpi.first.wpilibj.BuiltInAccelerometer;
```

Comment:
Why are we using the BuiltInAccelerometer instead of the Pigeon?

Reply:
more accurate when using acceleration in all 3 directions